src/arch/amdgpu/vega/insts/smem.cc

```diff
@@ -950,10 +951,26 @@ namespace VegaISA
     void
     Inst_SMEM__S_MEMTIME::execute(GPUDynInstPtr gpuDynInst)
     {
+        Wavefront *wf = gpuDynInst->wavefront();
+        gpuDynInst->execUnitId = wf->execUnitId;
+        gpuDynInst->latency.init(gpuDynInst->computeUnit());
+        gpuDynInst->latency.set(gpuDynInst->computeUnit()->clockPeriod()*41);
```

Comment:
Please add an input arg to make this configurable, instead of using 41 as a magic number.